run/django/README.md

```diff
@@ -0,0 +1,25 @@
+# Getting started with Django on Google Cloud Platform on Cloud Run (fully managed)
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=run/django/README.md
+
+This repository is an example of how to run a [Django](https://www.djangoproject.com/) 
+app on Google Cloud Run (fully managed). 
+
+The Django application used in this tutorial is the [Writing your first Django app](https://docs.djangoproject.com/en/3.0/#first-steps), after completing [Part 1](https://docs.djangoproject.com/en/3.0/intro/tutorial01/) and [Part 2](https://docs.djangoproject.com/en/3.0/intro/tutorial02/)).
```

Comment:
Are we 100% sure that Google authored this code and we're able to claim it under this repo's license? It seems like this tutorial should be hosted on Django's website/repo and not here.

Reply:
I followed the pattern from the [kubernetes](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/kubernetes_engine/django_tutorial) and [appengine](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/appengine/flexible/django_cloudsql) samples.